frontend/packages/knative-plugin/locales/ko/knative-plugin.json

```diff
@@ -2,7 +2,6 @@
   "Functions": "함수",
   "Event Source": "이벤트 소스",
   "Create an Event source to register interest in a class of events from a particular system": "이벤트 소스를 생성하여 특정 시스템에서 관심있는 이벤트 클래스에 등록합니다.",
-  "Import from git": "git에서 가져 오기",
```

Comment:
Does it get removed by running `yarn i18n` or do you remove it manually?

Reply:
I removed this manually because if you look at the file there is already an "Import from Git" key